odata-json-format/16 Bound Function.md

```diff
@@ -337,6 +337,11 @@ how to interpret the omitted parameter. Note: a nullable non-binding
 parameter is equivalent to being annotated as optional with a default
 value of `null`.
 
+Non-binding collection-valued parameters MAY be omitted from the request body regardless
+of nullability or optionality. Unless a `Core.OptionalValue` with `DefaultValue` is annotated,
+the interpretation of such an omitted parameter is up to the service,
+this includes assuming an empty collection or reporting an error.
```

Comment:
Note: This allows a service to treat an absent mandatory parameter of type `Collection(Edm.Int16)` like `[1,2]`.